lib/vmdb/loggers.rb

```diff
@@ -30,20 +30,9 @@ def self.apply_config(config)
     end
 
     def self.create_logger(log_file_name, logger_class = VMDBLogger)
-      log_file = ManageIQ.root.join("log", log_file_name)
-      progname = File.basename(log_file_name, ".*")
-
-      logger_class.new(log_file, progname: progname).tap do |logger|
-        broadcast_logger = create_broadcast_logger
-        if broadcast_logger
-          logger.extend(ActiveSupport::Logger.broadcast(broadcast_logger))
-          broadcast_logger.progname = progname
-
-          # HACK: In order to access the broadcast logger in test, we inject it
-          #   as an instance var.
-          logger.instance_variable_set(:@broadcast_logger, broadcast_logger) if Rails.env.test?
-        end
-      end
+      create_container_logger(log_file_name, logger_class) ||
+        create_journald_logger(log_file_name, logger_class) ||
+        create_file_logger(log_file_name, logger_class)
```

Comment:
Not a big fan of the `||` conditional logic here.
Would prefer just going back to the `if` from 'create_raw_container_logger` to here and then calling `create_wrapper_logger(log_file_name, logger_class, create_raw_container_logger)` directly.